content/articles/2018-05-ggplot2-2-3-0.Rmd

```diff
@@ -0,0 +1,126 @@
+---
+title: ggplot2 2.3.0 — upcoming release
+author: Mara Averick
+date: '2018-05-18'
+slug: ggplot2-2-3-0
+description: >
+    What you need to know about upcoming changes for ggplot2 2.3.0.
+categories: [package]
+photo:
+  url: https://unsplash.com/photos/OhQhkkVJYhI
+  author: chuttersnap
+---
+```{r setup, include = FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>", dpi = 300, fig.retina = 2)
+```
+
+A new release of ggplot2 (2.3.0) is on the horizon, and we'd love to have people 
+try it out, give us feedback, and [report issues](https://github.com/tidyverse/ggplot2/issues) 
+before we submit it to CRAN. This version is the culmination of over a year and 
+a half of development, not all of which will be captured here. So, please see [NEWS](https://github.com/tidyverse/ggplot2/blob/master/NEWS.md) for a fuller 
+account of updates and changes.
+
+In addition to highlighting a few features and improvements, we also want to 
+share a bit about our release-preparation process for ggplot2, which has over 
+1,700 reverse dependencies.
+
+## Breaking changes
+
+### Our process
+
+Regression testing, visual and otherwise, allows us to keep track of changes
```

Comment:
The first "you" here was a bit hard to parse for me. Maybe make it "we"? Or otherwise change the lead-in.